examples/platforms/simulation/radio.c

```diff
@@ -356,8 +375,24 @@ static void initFds(void)
 void platformRadioInit(void)
 {
 #if OPENTHREAD_SIMULATION_VIRTUAL_TIME == 0
+    char *base;
     char *offset;
 
+    base = getenv("PORT_BASE");
```

Comment:
nit: can we use a different name for `PORT_BASE`? When I saw 'base' alone, I thought it was about the radix. This is just a heads up. 
 No need to change it in this PR.